packages/constants/compile-constants.ts

```diff
@@ -0,0 +1,100 @@
+import fs from 'fs'
+import path from 'path'
+import { run } from '../cli/utils'
+
+const DATA_PATH = path.join(__dirname, 'data')
+const INDEX_PAYH = path.join(__dirname, 'src', 'index.ts')
+
+const files = fs.readdirSync(DATA_PATH)
```

Comment:
Is the order stable? IMO you should sort them.

Reply:
The order is doesn't matter.